shared/constants/tracker2.tsx

```diff
@@ -21,20 +21,20 @@ export const makeDetails = (): Types.Details => ({
   state: 'error',
   teamShowcase: emptyArray,
   username: '',
-})
+}
 
-export const makeNonUserDetails = (): Types.NonUserDetails => ({
+export const noNonUserDetails: Types.NonUserDetails = {
   assertionKey: '',
   assertionValue: '',
   description: '',
   siteIcon: emptyArray,
   siteIconFull: emptyArray,
   siteURL: '',
-})
+}
 
 export const generateGUIID = () => Math.floor(Math.random() * 0xfffffffffffff).toString(16)
 
-export const makeAssertion = (): Types.Assertion => ({
+export const noAssertion: Types.Assertion = {
```

Comment:
If nothing is using these `noAssertion`, etc, why have them?

Reply:
talked elsewhere, they're used to give default values in some places, the `make*` functions really weren't needed however.